scripts/deploy/config/databend-query-node-1.toml

```diff
@@ -29,7 +30,7 @@ http_handler_port = 8001
 cluster_id = "test_cluster"
 
 [log]
-log_level = "ERROR"
+log_level = "INFO"
```

Comment:
`INFO`  will prints many info to the CI log

Reply:
forget to discard change..now resolved.